README.md

```diff
@@ -1,6 +1,8 @@
 # pycloudlib
 
-[![Build Status](https://travis-ci.com/canonical/pycloudlib.svg?branch=master)](https://travis-ci.com/canonical/pycloudlib)
+[![Pycloudlib CI](https://github.com/canonical/pycloudlib/actions/workflows/ci.yaml/badge.svg)](https://github.com/canonical/pycloudlib/actions/workflows/ci.yaml)
```

Comment:
Use main_check.yaml instead of ci.yaml since ci.yaml is only for PRs.
```suggestion
[![Pycloudlib Main CI](https://github.com/canonical/pycloudlib/actions/workflows/main_check.yaml/badge.svg)](https://github.com/canonical/pycloudlib/actions/workflows/main_check.yaml)
```